client/src/components/v3/status/WMStatus.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { addArgs, createStory } from 'utils/stories';
+import { SensorDataT, SensorsT } from 'types/data';
+import WMStatus, { WMStatusProps } from './WMStatus';
+
+export default {
+  component: WMStatus,
+  title: 'components/v3/status/WirelessModuleStatus',
+};
+
+const Template = addArgs<WMStatusProps>((props) => <WMStatus {...props} />);
+
+const sensorData = (type: string, value: SensorsT): SensorDataT => ({
+  type,
+  value,
+});
+
+/* ----------------------------------- Stories ----------------------------------- */
+
+export const Disconnected = createStory(Template, {
+  moduleName: 'Front WM',
+  isOnline: null,
+  data: [],
+});
+
+export const Offline = createStory(Template, {
+  moduleName: 'Front WM',
+  isOnline: false,
+  data: [],
+});
```

Comment:
How do you differentiate disconnected vs offline? They sound the same to me. (if they are, consider making `isOnline` strictly boolean)

Reply:
I think it's something like this

- Disconnected: Not connected to MQTT
- Offline: `/stop`
- Online: `/start`

@Blake-Haydon confirm?

Reply:
I think this is a bit misleading. Would prefer if online meant that the WM is reachable at all, otherwise I can see people getting confused as to why the sensors show up as offline when everything is set up properly.

Perhaps in the section that shows the raw sensor data we could show "sensor is not publishing data" when this is the case, to get the same functionality you've got currently.

Reply:
I've removed the "Disconnected" state to be more consistent with the camera status, as well as it not being very clearly defined. Could add it back later if there is a need.
abc8b18